jina/serve/runtimes/gateway/graph/topology_graph.py

```diff
@@ -79,7 +79,7 @@ async def _wait_previous_and_send(
                             DataRequestHandler.reduce_requests(self.parts_to_send)
                         ]
 
-                    resp, metadata = await connection_pool.send_requests_once(
+                    (resp, metadata,) = await connection_pool.send_requests_once(
```

Comment:
why this change?

Reply:
No idea, I would say it happened which black or something. Will fix it